README.md

```diff
@@ -1 +1,11 @@
-# mentors
\ No newline at end of file
+# Exercism Mentors
+
+We are always looking for more mentors to help new people ramp up in the programming languages that Exercism supports.
+
+Read more about mentorship at Exercism on [mentors.exercism.io](https://mentors.exercism.io/).
+
+## How many mentors do we need?
+
+Well, that depends. For any given track, the minimum number of active mentors is *five*. This ensures that the burden is never too great for a single mentor, even when life gets busy for a bit, or you go on vacation. Beyond this, we need an extra mentor for every ten-twelve additional submissions per week.
+
+For reference, the most active track at the moment is the Python track, with about 150 submissions per day. This equates roughly to a need for about 90 active mentors.
```

Comment:
I think we should say "This equates roughly to a need for about 90 active mentors, each giving about 1 hour per week" or similar. This currently feels like we need 90 mentors on every night, which sounds intimidating.

Reply:
Oooh, right! Yeah that's much better.